doc/prelimsLinux.rst

```diff
@@ -0,0 +1,222 @@
+Preliminary Steps for Linux Distributions
+=========================================
+
+1) Install Python 3.6+
+----------------------
+Check your current python version(s) by entering the following commands into the command line: 
+
+.. code-block:: shell
+
+   python --version
+   python3 --version
+   python3.6 --version
+
+If Python 3.6+ is not present, follow these instructions depending on your operating system:
+
+Running a **Linux** distribution, Python must be compiled. Download and compile the most recent Python release with the following commands (find the link to the correct .tar file `here <https://www.python.org/downloads/>`__):
+
+.. note::
+
+   If CMake is not already on your machine see step 3a to install it.
+
+.. code-block:: shell
+   
+   wget https://www.python.org/ftp/python/3.6.1/Python-3.6.1.tar.xz
+   sudo tar xzf Python-3.6.1.tar.xz
+   cd Python-3.6.1
+   ./configure       # sudo might be required
+   make altinstall   # sudo might be required
+
+Some **newer Linux** distributions may be able to install Python 3.6+ using apt-get.
```

Comment:
Specifically, this only works for Ubuntu or other distros that use `apt`, like Mint. It wouldn't be correct for, say, Fedora, which uses `dnf`. Maybe just say to use your package manager.